napari/layers/points/points.py

```diff
@@ -862,79 +775,91 @@ def edge_color_mode(self) -> str:
 
         COLORMAP allows color to be set via a color map over an attribute
         """
-        return str(self._edge_color_mode)
+        return self._edge.mode
 
     @edge_color_mode.setter
     def edge_color_mode(self, edge_color_mode: Union[str, ColorMode]):
         self._set_color_mode(edge_color_mode, 'edge')
 
+    @property
+    def face(self) -> ColorManager:
+        """The ColorManager object that coordinates the face color of each point"""
+        return self._face
```

Comment:
I think we should wait on this for the next PR when we add all the deprecations for the old attributes, but I love it otherwise.

Reply:
Sounds good. I'll remove it.